src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6264,7 +6269,12 @@ static void measureStyle(XmlWriter& xml, Attributes& attr, const Measure* const
     const Measure* mmR1 = m->coveringMMRestOrThis();
     if (m != mmR1 && m == mmR1->mmRestFirst()) {
         attr.doAttr(xml, true);
-        xml.startElement("measure-style");
+        String styleTag = u"measure-style";
+        if (m->score()->style().styleB(Sid::oldStyleMultiMeasureRests)
+            && mmR1->mmRestCount() < m->score()->style().styleI(Sid::mmRestOldStyleMaxMeasures)) {
+            styleTag += u" use-symbols=\"yes\"";
```

Comment:
Looks like this needs to be an attribute of the `multiple-rest` tag, instead of of the `measure-style` tag: https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/multiple-rest/

Reply:
Absolutely correct, thanks!